### ISSUE TITLE
If use /is reset 1, get this error

### ISSUE THREAD

**Issue by NycuRO** [larryTheCoder/ASkyBlock-Nukkit#12]
2017-3-4 12:52:26 [EMERGENCY] Exception happened while ticking server
2017-3-4 12:52:26 [ALERT] java.lang.StackOverflowError
	at com.larryTheCoder.command.SubCommand.getPrefix(SubCommand.java:54)
	at com.larryTheCoder.command.SubCommand.getPrefix(SubCommand.java:54)
	at com.larryTheCoder.command.SubCommand.getPrefix(SubCommand.java:54)
	at com.larryTheCoder.command.SubCommand.getPrefix(SubCommand.java:54)
	at com.larryTheCoder.command.SubCommand.getPrefix(SubCommand.java:54)
	at com.larryTheCoder.command.SubCommand.getPrefix(SubCommand.java:54)
	at com.larryTheCoder.command.SubCommand.getPrefix(SubCommand.java:54)

*** AND GET CRASH SERVER WITH THIS ERROR ***

2017-3-4 12:52:26 [ALERT] 
"threadDeathWatcher-3-1" Id=24 TIMED_WAITING
	at java.lang.Thread.sleep(Native Method)
	at io.netty.util.ThreadDeathWatcher$Watcher.run(ThreadDeathWatcher.java:150)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
	at java.lang.Thread.run(Thread.java:745)

"Nukkit Asynchronous Task Handler #5" Id=23 WAITING on java.util.concurrent.SynchronousQueue$TransferStack@548e76f1
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@548e76f1
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:458)
	at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:362)
	at java.util.concurrent.SynchronousQueue.take(SynchronousQueue.java:924)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	...

"Nukkit Asynchronous Task Handler #4" Id=22 WAITING on java.util.concurrent.SynchronousQueue$TransferStack@548e76f1
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@548e76f1
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:458)
	at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:362)
	at java.util.concurrent.SynchronousQueue.take(SynchronousQueue.java:924)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	...

"Nukkit Asynchronous Task Handler #3" Id=21 WAITING on java.util.concurrent.SynchronousQueue$TransferStack@548e76f1
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@548e76f1
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:458)
	at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:362)
	at java.util.concurrent.SynchronousQueue.take(SynchronousQueue.java:924)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	...

"Nukkit Asynchronous Task Handler #2" Id=20 WAITING on java.util.concurrent.SynchronousQueue$TransferStack@548e76f1
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@548e76f1
	at java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
	at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:458)
	at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:362)
	at java.util.concurrent.SynchronousQueue.take(SynchronousQueue.java:924)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1067)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	...

"nioEventLoopGroup-2-1" Id=18 RUNNABLE (in native)
	at sun.nio.ch.EPollArrayWrapper.epollWait(Native Method)
	at sun.nio.ch.EPollArrayWrapper.poll(EPollArrayWrapper.java:269)
	at sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:93)
	at sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:86)
	-  locked io.netty.channel.nio.SelectedSelectionKeySet@5aabbb29
	-  locked java.util.Collections$UnmodifiableSet@72c927f1
	-  locked sun.nio.ch.EPollSelectorImpl@1ac85b0c
	at sun.nio.ch.SelectorImpl.select(SelectorImpl.java:97)
	at io.netty.channel.nio.NioEventLoop.select(NioEventLoop.java:759)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:400)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:873)
	...

"Abandoned connection cleanup thread" Id=17 TIMED_WAITING on java.lang.ref.ReferenceQueue$Lock@3dd69f5a
	at java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@3dd69f5a
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:143)
	at com.mysql.jdbc.AbandonedConnectionCleanupThread.run(AbandonedConnectionCleanupThread.java:43)

"RakNet Thread #14" Id=14 TIMED_WAITING
	at java.lang.Thread.sleep(Native Method)
	at cn.nukkit.raknet.server.SessionManager.tickProcessor(SessionManager.java:96)
	at cn.nukkit.raknet.server.SessionManager.run(SessionManager.java:70)
	at cn.nukkit.raknet.server.SessionManager.<init>(SessionManager.java:58)
	at cn.nukkit.raknet.server.RakNetServer.run(RakNetServer.java:101)

"Console" Id=9 RUNNABLE (in native)
	at java.io.FileInputStream.read0(Native Method)
	at java.io.FileInputStream.read(FileInputStream.java:207)
	at jline.internal.NonBlockingInputStream.read(NonBlockingInputStream.java:166)
	-  locked jline.internal.NonBlockingInputStream@3aa3193a
	at jline.internal.NonBlockingInputStream.read(NonBlockingInputStream.java:135)
	at jline.internal.NonBlockingInputStream.read(NonBlockingInputStream.java:243)
	at jline.internal.InputStreamReader.read(InputStreamReader.java:257)
	-  locked jline.internal.NonBlockingInputStream@3aa3193a
	at jline.internal.InputStreamReader.read(InputStreamReader.java:194)
	-  locked jline.internal.NonBlockingInputStream@3aa3193a
	at jline.console.ConsoleReader.readCharacter(ConsoleReader.java:2143)
	...

"NonBlockingInputStreamThread" Id=13 WAITING on jline.internal.NonBlockingInputStream@3aa3193a owned by "Console" Id=9
	at java.lang.Object.wait(Native Method)
	-  waiting on jline.internal.NonBlockingInputStream@3aa3193a
	at jline.internal.NonBlockingInputStream.run(NonBlockingInputStream.java:275)
	at java.lang.Thread.run(Thread.java:745)

"process reaper" Id=11 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@1ee4730
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@1ee4730
	at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
	at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:460)
	at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:362)
	at java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:941)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1066)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	...

"process reaper" Id=10 TIMED_WAITING on java.util.concurrent.SynchronousQueue$TransferStack@1ee4730
	at sun.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.SynchronousQueue$TransferStack@1ee4730
	at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:215)
	at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:460)
	at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:362)
	at java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:941)
	at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1066)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1127)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	...

"Thread-0" Id=8 RUNNABLE (in native)
	at java.lang.UNIXProcess.forkAndExec(Native Method)
	at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
	at java.lang.ProcessImpl.start(ProcessImpl.java:134)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	at jline.internal.TerminalLineSettings.exec(TerminalLineSettings.java:293)
	at jline.internal.TerminalLineSettings.stty(TerminalLineSettings.java:282)
	at jline.internal.TerminalLineSettings.get(TerminalLineSettings.java:143)
	at jline.internal.TerminalLineSettings.fetchConfig(TerminalLineSettings.java:190)
	...

"Signal Dispatcher" Id=4 RUNNABLE

"Finalizer" Id=3 WAITING on java.lang.ref.ReferenceQueue$Lock@5003041b
	at java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.ReferenceQueue$Lock@5003041b
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:143)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:164)
	at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:209)

"Reference Handler" Id=2 WAITING on java.lang.ref.Reference$Lock@724bade8
	at java.lang.Object.wait(Native Method)
	-  waiting on java.lang.ref.Reference$Lock@724bade8
	at java.lang.Object.wait(Object.java:502)
	at java.lang.ref.Reference.tryHandlePending(Reference.java:191)
	at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:153)

"main" Id=1 RUNNABLE
	at sun.management.ThreadImpl.dumpThreads0(Native Method)
	at sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:454)
	at cn.nukkit.utils.Utils.getAllThreadDumps(Utils.java:110)
	at cn.nukkit.Server.tickProcessor(Server.java:816)
	at cn.nukkit.Server.start(Server.java:781)
	at cn.nukkit.Server.<init>(Server.java:466)
	at cn.nukkit.Nukkit.main(Nukkit.java:68)

2017-3-4 12:52:26 [INFO] Disabling EconomyAPI v1.0.1
2017-3-4 12:52:26 [INFO] Disabling DbLib v0.2.1
2017-3-4 12:52:26 [INFO] Disabling ASkyBlock v0.2.6

@larryTheCoder

**Comment by larryTheCoder** [comment]
I am having an issue with my computer now my IDE crashes and crashes do you have any idea / any IDE that you know (maybe that can help me improve my Java Skills?)

@NycuRO

**Comment by NycuRO** [comment]
Move files project on other location. Dezistall IDE and Java and install again, @larryTheCoder

**Comment by NycuRO** [comment]
@larryTheCoder Ready?

**Comment by larryTheCoder** [comment]
FIxed now my PC has a problem it cant boot this will be a major problem of my coding situation so this will be postponed until next month

**Comment by NycuRO** [comment]
@larryTheCoder Update larry?????????????????

**Comment by iDirtniverse** [comment]
@NycuRo give him sometime, porting it isn't as easy as you seen

**Comment by NycuRO** [comment]
@iDirtniverse He is inactive.

**Comment by iDirtniverse** [comment]
@larryTheCoder :( Are you busy?
